src/services/chats.js

```diff
@@ -226,6 +252,19 @@ const uploadChatImgDoneHandler = async (req, res) => {
   }
 };
 
+/**
+ * 주어진 유저가 주어진 방에 참여하는 중인지 확인합니다.
+ * @summary 채팅 load/send 관련 api에서 검증을 위하여 함수로 분리하였습니다.
+ * @param {string} userId - 확인하고픈 user의 userId입니다.
+ * @param {string} roomId - 참여하는지 확인하고픈 방의 roomId입니다.
+ * @return {Promise<Boolean>} userId가 방에 포함되어 있다면 true, 그 외의 경우 false를 반환합니다.
+ */
+const isUserInRoom = async (userId, roomId) => {
+  const { part } = await roomModel.findById(roomId);
+  if (!part) return false;
+  return part.map((participant) => participant.user).includes(userId);
+};
+
```

Comment:
```suggestion
const isUserInRoom = async (userId, roomId) => {
  const room = await roomModel.findOne({ _id: roomId, part: { userId } }, "_id");
  return !!room;
};
```

한 번의 DB 쿼리 연산으로 user가 방에 포함되어 있는지 여부를 확인할 수 있을 것 같습니다.

Reply:
part가 object가 아니라 object의 배열이어도 저렇게 해서 찾을 수 있나요? @14KGun